traitsui/editor.py

```diff
@@ -129,6 +135,15 @@ class Editor(HasPrivateTraits):
     #: The current editor invalid state status:
     invalid = Bool(False)
 
+    #: A set to track values being updated to prevent infinite recursion.
+    _no_trait_update = Set()
```

Comment:
From the code, it looks as though this is a set of strings. Is that correct? If so, please could you add that information to the traits doc-comment?